### ISSUE TITLE
Transparency doesn't clean the previous screen when refreshed

### ISSUE THREAD

**Issue by BAKFR** [Cqoicebordel/webslice-plasmoid#15]
If the transparency option is activated, the plasmoid doesn't clean the screen on refresh.

It means when we navigate on other pages or when we refresh the page, the new text is displayed on top of the previous one, and both contents overlaps.

Versions of KDE on my system:
Plasma: 5.13.5
KDE frameworks: 5.50
Qt: 5.11.1

**Comment by Cqoicebordel** [comment]
Yes, I saw that on my own websliced websites. I had to use a background-color to keep using them.
For me, it was caused from the upgrade from Kubuntu 17.10 to 18.04.
I have no idea how to fix it, sadly.

The possibility of transparency was an experimental setting, as seen on https://rschroll.github.io/beru/2013/08/21/qtwebview.experimental.html
I think a modification of the C++ code of the webview is necessary, in Qt, which is beyond my reach right now.

Let's just hope it'll get fixed :/

**Comment by Cqoicebordel** [comment]
Thanks !
